image-loader/app/model/Uploader.scala

```diff
@@ -232,7 +232,7 @@ object Uploader extends GridLogging {
 
   private def toFileMetadata(f: File, imageId: String, mimeType: Option[MimeType]): Future[FileMetadata] = {
     mimeType match {
-      case Some(Png | Tiff) => FileMetadataReader.fromIPTCHeadersWithColorInfo(f, imageId, mimeType.get)
+      case Some(Png | Tiff | Jpeg) => FileMetadataReader.fromIPTCHeadersWithColorInfo(f, imageId, mimeType.get)
       case _ => FileMetadataReader.fromIPTCHeaders(f, imageId)
```

Comment:
if we're happy with this PR, a note to remember to clean up as the `case _` branch here is dead

Reply:
not dead, if `mimeType` is a `None`

Reply:
`fromIPTCHeaders` might as well be dead? But let’s pretend we don’t know, because I don’t :-).